materialrangebar/src/main/java/com/appyvet/materialrangebar/RangeBar.java

```diff
@@ -920,6 +969,44 @@ public void setPinRadius(float pinRadius) {
         createPins();
     }
 
+    /**
+     * Sets left selector circle color
+     *
+     * @param mCircleColorLeft
+     */
+    public void setSelectorColorLeft(int mCircleColorLeft) {
+        this.mCircleColorLeft = mCircleColorLeft;
+        createPins();
+    }
+
+    /**
+     * Sets Right selector circle color
+     *
+     * @param mCircleColorRight
+     */
+    public void setSelectorColorRight(int mCircleColorRight) {
```

Comment:
`setSelectorColorRight ` should be renamed to `setRightSelectorColor` to be consistent with the xml properties